.github/workflows/flake8.yml

```diff
@@ -0,0 +1,26 @@
+name: Code Quality
```

Comment:
I"m not sure running flake8u in this is worth it -- this isn't really a code project.

It would be nice to clean up all the code samples -- some of them are pretty old, but I dont hink we need to run a linter on every merge.